### ISSUE TITLE
softmax_cross_entropy_with_logits function in train_baseline.py

### ISSUE THREAD

**Issue by xuweijiezds** [carlini/breaking_defensive_distillation#1]
breaking_defensive_distillation/train_baseline.py line34: Should we switch the parameter logits/TRAIN_TEMP and train_ys in function tf.nn.softmax_cross_entropy_with_logits?

**Comment by carlini** [comment]
The code is now compatible with TF 1.0 and later as of c8e27c96af2a789500e2e1cb80af03c947c9518a.
